src/components/schemas/department.yaml

```diff
@@ -58,6 +63,10 @@ department_out_embed:
       type: string
       nullable: true
       example: Outbound Sales
+    display_name:
```

Comment:
I think we should mention that if the sub-department is null, then it will be `Sales` only